sql/core/pom.xml

```diff
@@ -87,6 +87,16 @@
     </dependency>
 
     <dependency>
+      <groupId>org.apache.orc</groupId>
+      <artifactId>orc-core</artifactId>
+      <classifier>${orc.classifier}</classifier>
```

Comment:
Yes. sbt understands `classifier` in pom file. Also, we can use classifier in sbt build file, too.